### ISSUE TITLE
Cannot start Postgres.app "psql: FATAL:  no pg_hba.conf"

### ISSUE THREAD

**Issue by tsurantino** [PostgresApp/PostgresApp#176]
Hi,

I am in a bit of a rabbit hole with the Postgres instances installed on my Mac OS X Mavericks machine. I've dug up an project to fix some bugs, but due to a variety if issues, had to reinstall my virtualenvironment. 

For whatever reason, I had trouble accessing my Postgres databases. At some point I realized that I needed to have one type of Postgres installation running. I opted to have PostgresApp for simplicity. I've tried uninstalling the Homebrew version, but I am not sure how successful I was because I am now consistently encountering the following error:

psql: FATAL:  no pg_hba.conf entry for host "[local]", user "arturtsurkan", database "arturtsurkan", SSL off

This is what my pg_hba.conf file looks like:

```
# TYPE  DATABASE        USER            ADDRESS                 METHOD

# "local" is for Unix domain socket connections only
local   all             all                                     trust
# IPv4 local connections:
host    all             all             127.0.0.1/32            trust
# IPv6 local connections:
host    all             all             ::1/128                 trust
# Allow replication connections from localhost, by a user with the
# replication privilege.
local   replication     arturtsurkan                                trust
host    replication     arturtsurkan        127.0.0.1/32            trust
host    replication     arturtsurkan        ::1/128                 trust
```

This file is located at /Users/arturtsurkan/Library/Application Support/Postgres/var

Basically, I would appreciate any help in trying to reset my Postgres installation so that I can use the Postgres.app. I am trying to get Postgres to run with Django on VirtualEnv, if that helps at all.
